run-warc-tests.sh

```diff
@@ -4,18 +4,24 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
 DATE=${DATE:-$(date +%Y-%m-%d)}
 
-ARCHIVE_DIR=$(dirname $(readlink -f $0))
-ARCHIVES="${ARCHIVE_DIR}/ARCHIVES"
+SCRIPT_FILE=$(readlink -f "$0")
+
+SCRIPT_DIR=$(dirname "${SCRIPT_FILE}")
+ARCHIVES="${SCRIPT_DIR}/ARCHIVES"
 
-SERVO_DIR=$(readlink -f ${SERVO_DIR:-../servo})
-SERVO="${SERVO_DIR}/mach run -r -z \
-  --userscripts ${ARCHIVE_DIR}/user-agent-js \
-  --certificate-path ${ARCHIVE_DIR}/proxy-certs/pywb-ca.pem \
-  --pref dom.testperf.enabled"
+SERVO_DIR=$(readlink -f "${SERVO_DIR:-../servo}")
+SERVO_CMD=("${SERVO_DIR}/mach" run -r -z
+  --userscripts "${SCRIPT_DIR}/user-agent-js" \
+  --certificate-path "${SCRIPT_DIR}/proxy-certs/pywb-ca.pem" \
+  --pref dom.testperf.enabled)
```

Comment:
Backslashes are not needed for line continuation inside an array, please remove them.
A formatting nit: I like block formatting and quoting all array members, e.g.
```
SERVO_CMD=(
    "${SERVO_DIR}/mach" 'run' '-r' '-z'
    ...
    '--pref' 'dom.testperf.enabled'
)
```

Reply:
Quoted the strings and removed the backslashes.